pkg/apis/crd/v1alpha1/types.go

```diff
@@ -17,81 +17,9 @@ package v1alpha1
 import (
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
-type TraceflowPhase string
```

Comment:
could you create another PR to cleanup these dead code?

Reply:
https://github.com/antrea-io/antrea/pull/6667/

created this MR to cleanup the code .will reset the changes in this MR soon.